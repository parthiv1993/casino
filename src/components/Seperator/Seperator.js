import styled from 'styled-components';

const Seperator = styled.hr`
  width: 100%;
`;

export default Seperator;
