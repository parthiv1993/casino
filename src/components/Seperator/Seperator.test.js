import { render } from '@testing-library/react';
import Seperator from './Seperator';

describe('Seperator', () => {
  it('should match snapshot', () => {
    const { container } = render(<Seperator />);
    expect(container).toMatchSnapshot();
  });
});
