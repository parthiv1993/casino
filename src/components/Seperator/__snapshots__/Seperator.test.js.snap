// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Seperator should match snapshot 1`] = `
.c0 {
  width: 100%;
}

<div>
  <hr
    class="c0"
  />
</div>
`;
